app-text/robodoc/robodoc-4.99.41.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="Automating Software Documentation"
+HOMEPAGE="http://www.xs4all.nl/~rfsber/Robo/robodoc.html"
+SRC_URI="https://rfsber.home.xs4all.nl/Robo/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples"
+
+RDEPEND=">=dev-util/ctags-5.3.1"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	# move and sed due to configure.in and autotoolsversion (still only a warning)
+	# For more details see:
+	# https://bugs.gentoo.org/426262
+	mv ./configure.in ./configure.ac
+	sed -i -e 's/^dnl/\#/g' ./configure.ac
```

Comment:
Here, too.

Reply:
The only stable version kept has no need for the configure.ac fix. This was fixed upstream with the stable release of 4.99.43